packages/analytics/src/integrations/Omnitracking/__tests__/__snapshots__/Omnitracking.test.ts.snap

```diff
@@ -100,6 +100,16 @@ Object {
 }
 `;
 
+exports[`Omnitracking track events definitions \`Interact Content\` return should match the snapshot 1`] = `
+Object {
+  "actionArea": undefined,
```

Comment:
Can you add a value on the mock data to ensure this snapshot property contains a valid value, please?